metadata/Dev-Corner.yml

```diff
@@ -121,6 +121,35 @@ material:
     - "Help yourself by using the available web plugins and record an interactive tour"
 
 -
+    title: "Galaxy Interactive Environments"
+    type: "tutorial"
+    name: "interactive_environments"
+    zenodo_link: ""
+    galaxy_tour: ""
+    hands_on: "yes"
```

Comment:
Do you plan to add an hands-on specific to the IE? Because here, a link to an hands-on will be generated...